xtask/src/fpga/configurations.rs

```diff
@@ -153,8 +153,11 @@ impl<'a> ActionHandler<'a> for Subsystem {
         Ok(())
     }
 
-    fn build_test(&self, _args: &'a BuildTestArgs<'a>) -> Result<()> {
-        let mut base_cmd = build_base_docker_command(None::<String>)?;
+    fn build_test(&self, args: &'a BuildTestArgs<'a>) -> Result<()> {
+        // Delete the file if it exists. Sometimes the docker build fails silently. This will force
+        // the rsync to fail in those cases.
+        let _ = std::fs::remove_file("caliptra-test-binaries.tar.zst");
```

Comment:
Should this be on line 92? Then all the `build_test` types will delete the test binaries.

Reply:
That's a good idea! I moved it to the higher level function.